### DISCUSSION TITLE
how commands work? (just curious lmao)

### DISCUSSION THREAD

**Discussion by RSChao** [Scythe-Anticheat/Scythe-Anticheat#111] [General]
i just wanted to know since appearently custom commands now arent as easy as old tutorials say, and i found nothing to make custom chat commands, except this addon. So if I knew how to separate args and cancel the message, id be very grateful ngl. Just curious tho

**Comment by ExtraBytez** [comment]
See the bedrock https://wiki.bedrock.dev you also might want to know some js
